source/uwp/Renderer/lib/XamlBuilder.cpp

```diff
@@ -153,6 +153,14 @@ AdaptiveNamespaceStart
             THROW_IF_FAILED(MakeAndInitialize<AdaptiveRenderArgs>(&bodyRenderArgs, containerStyle, childElementContainerAsFE.Get()));
             BuildPanelChildren(body.Get(), bodyElementContainer.Get(), renderContext, bodyRenderArgs.Get(), [](IUIElement*) {});
 
+            ComPtr<IAdaptiveActionElement> selectAction;
+            THROW_IF_FAILED(adaptiveCard->get_SelectAction(&selectAction));
+
+            // Create a new IUIElement pointer to house the root element decorated with select action
+            ComPtr<IUIElement> rootSelectActionElement;
+            HandleSelectAction(nullptr, selectAction.Get(), renderContext, rootElement.Get(), SupportsInteractivity(hostConfig.Get()), true, &rootSelectActionElement);
+            THROW_IF_FAILED(rootSelectActionElement.As(&childElementContainerAsFE));
```

Comment:
test codeflow

Reply:
test resolved comment

---
In reply to: [183123853](https://github.com/Microsoft/AdaptiveCards/pull/1396#discussion_r183123853) [](ancestors = 183123853)